pom.xml

```diff
@@ -0,0 +1,75 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.github.wechaty</groupId>
+  <artifactId>wechaty-grpc-proto</artifactId>
```

Comment:
Is this the package name of our Java GRPC module?

If so, I'd like to rename it from `wechaty-grpc-proto` to just `grpc`, so that it's path will be `io.github.wechaty/grpc`.

Please feel free to correct me if I'm not right.